src/_posts/languages/php/2000-01-01-start.md

```diff
@@ -64,57 +68,50 @@ The default PHP version on both `scalingo-20` and `scalingo-22` is the latest
 ```
 
 {% note %}
-  You should not specify a precise version such as `8.2.3` or you would miss
-  important updates. Instead, you should specify `~8.2.3` to install a version
-  `>=8.2.3 and <8.3.0` or specify `~8.2` to install a version `>=8.2 and
-  <9.0.0`.
+You should not specify a precise version such as `8.2.3` or you would miss
+important updates. Instead, you should specify `~8.2.3` to install a version
+`>=8.2.3 and <8.3.0` or specify `~8.2` to install a version `>=8.2 and <9.0.0`.
 
-  Further details about version constraints can be found in the
-  [Composer documentation](https://getcomposer.org/doc/articles/versions.md#writing-version-constraints)
+Further details about version constraints can be found in the
+[Composer documentation](https://getcomposer.org/doc/articles/versions.md#writing-version-constraints)
 {% endnote %}
 
+
 ## PHP Extensions and Dependencies
 
 See our dedicated documentation pages:
 
 - [Working with PHP Dependencies]({% post_url languages/php/2000-01-01-dependencies %})
 - [Working with PHP Extensions]({% post_url languages/php/2000-01-01-extensions %})
 
-## Officially Supported Frameworks
```

Comment:
suggestion: as you remove this section which is saying that we are compatible with many frameworks.
I think you can add a "note" section in the "Framework" section below which say that the list is not an exhaustive one.
WDYT?

Reply:
The previous paragraph was a bit misleading IMHO:

> Scalingo supports out-of-the-box many well-known frameworks. When using such frameworks, you have nothing special to configure in your `composer.json`, `git push` your code and everything will work!

While this is true for some frameworks, it's not for all of them. Some require little tweaks, like adding some Nginx configuration.

Hence replacing the paragraph with something less specific:

> Here is a non-exhaustive list of frameworks known to work on Scalingo:

I also suggest to remove the "Officially" from the title. I don't think it makes sense anymore.
And, by the way, the non-exhaustive property of the list is here :yum:

Reply:
Wut? o_O

I deleted this comment as I viewed this line:
> Here is a non-exhaustive list of frameworks known to work on Scalingo:

I'm maybe drunk or GitHub is...